access-control-app/int_test/cmr/access_control/int_test/permission_check_test.clj

```diff
@@ -171,55 +173,55 @@
                 "user2" ["read" "order"]))))
 
         (testing "acls granting access to collection by collection identifier"
-          (testing "by concept id"
+          (testing "by entry title"
             (create-acl
               {:group_permissions [{:permissions [:read]
                                     :user_type :guest}]
-               :catalog_item_identity {:name "coll2 guest read concept ids"
+               :catalog_item_identity {:name "coll2 guest read entry titles"
                                        :collection_applicable true
-                                       :collection_identifier {:concept_ids [coll2]}
+                                       :collection_identifier {:entry_titles ["coll2 entry title"]}
                                        :provider_id "PROV1"}})
-            (testing "for collection in ACL's concept ids"
-              (are3 [user permissions]
+            (testing "for collection in ACL's entry titles"
+              (are [user permissions]
```

Comment:
Isn't `are` deprecated? You should be able to do this with `are3` ... up to you :-)